src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,95 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import frc.robot.config.Config;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class ShooterSubsystem extends SubsystemBase {
+
+  private CANSparkMax m_shooter;
+  private CANPIDController m_pidController;
+  private CANEncoder m_encoder;
+
+  double kF = 0.00018;
```

Comment:
Can you make these constants? Also what are the values for, e.g. kF == SHOOTER_FUDGE_FACTOR?